src/module/vue/site-sheet.vue

```diff
@@ -299,7 +306,7 @@ async function randomDenizen() {
   )
   const idx = $actor?.data.data.denizens.indexOf(denizen)
   if (!denizen) throw new Error(`Rolled a ${result} but got no denizen???`)
-  await CONFIG.IRONSWORN.createIronswornDenizenChat({
+  await createIronswornDenizenChat({
```

Comment:
this one's *hella* minor, but - it occurs to me that the meaning of "IronswornDenizen" might be less than obvious to someone who isn't familiar with Delve (which, with Starforged now existing, isn't uncommon). probably better to err on the side of clarity: maybe it could be something like `DelveSiteDenizen`?

this has the possible fringe benefit of making it easier to search the codebase for delve-specific code (possible application: developing a user setting to toggle Delve content off, if they someone doesn't want the clutter).

now that I think about it: would it be worth standardizing the internal naming conventions to better distinguish what game something is for? i reckon the problem with using `Ironsworn` as the distinguishing label for "Classic" Ironsworn is that it's also the label used for all three games collectively.

i'm thinking a standard that goes something like...

* `Ironsworn*` indicates that something that applies across both *Ironsworn* and *Ironsworn: Starforged* (when it's necessary to make a distinction from e.g. FoundryVTT's version of something, like with `IronswornActor`). 
* `Classic*` indicates that something is exclusive to the original *Ironsworn*, but not exclusive to *Ironsworn: Delve*
* `Delve*` indicates that something is exclusive to *Ironsworn: Delve*
* `Starforged*` indicates that something is exclusive to *Ironsworn: Starforged*
* `SunderedIsles*` when it shows up, etc

Reply:
I like it.

Reply:
cool, i've opened a separate issue for that: #426